.env.production.sample

```diff
@@ -301,3 +301,10 @@ MAX_POLL_OPTION_CHARS=100
 # -----------------------
 IP_RETENTION_PERIOD=31556952
 SESSION_RETENTION_PERIOD=31556952
+
+# Local-only Emoji
+# **Always add old local-only emojis to the ALTERNATIVE_LOCAL_ONLY_EMOJI**,
+# otherwise old local-only messages will become public
```

Comment:
I don't think this is actually true.
local-only toots have a local_only attribute which is independent from the emoji.
The emoji is only added to local-only toots to have a distinction in third-party clients not supporting this feature.
And to be able to toot local-only toots via third-party clients.
Changing the emoji would only affect new toots.